apps/rush-lib/src/scripts/install-run.ts

```diff
@@ -437,13 +437,17 @@ export function installAndRun(
   console.log(os.EOL + statusMessage + os.EOL + statusMessageLine + os.EOL);
 
   const binPath: string = _getBinPath(packageInstallFolder, packageBinName);
+  const binFolderPath: string = path.resolve(packageInstallFolder, NODE_MODULES_FOLDER_NAME, '.bin');
   const result: childProcess.SpawnSyncReturns<Buffer>  = childProcess.spawnSync(
     binPath,
     packageBinArgs,
     {
       stdio: 'inherit',
       cwd: process.cwd(),
-      env: process.env
+      env: {
+        ...process.env,
+        PATH: `${binFolderPath}:${process.env.PATH}`
```

Comment:
@ghaiklor Could you test this on both Windows and Mac/Linux?

IIRC the shell path separator may be different.  https://nodejs.org/api/path.html#path_path_delimiter

Reply:
Wonderful catch, thanks, updated. Regard to testing, only Mac here.

Reply:
Old `install-run-rush.js`:

```
Starting "rush all"
sh: rush: command not found
```

New `install-run-rush.js`:

```
Starting "rush all"

Rush Multi-Project Build Tool 5.24.3 - https://rushjs.io
Node.js version is 14.4.0 (pre-LTS)

Starting "rush clean"
# ...
```